docs/releases/legacy-changes.md

```diff
@@ -26,4 +26,20 @@ More rational model/UX - `kops create cluster` just creates spec, `kops update c
 * Most commands accept positional arguments for the cluster name (you can specify `kops update cluster <name>`,
```

Comment:
Did not want to delete, as  I am not certain the information is on the release page.